suzieq/poller/worker/services/service.py

```diff
@@ -26,6 +26,8 @@
 HOLD_TIME_IN_MSECS = 60000
 # How many unsuccessful polls before marking records with active=false
 HYSTERESIS_FAILURE_CNT = 3
+# The boot timestamp time drift tolerance (in seconds)
+TIME_DRIFT_TOLERANCE = 30
```

Comment:
30s seems a lot. Why this high? This assumes a default polling period of 1 min?

Reply:
The polling period is not really involved, since the boot timestamp is computed when we initialize the the device data, after we connect (or reconnect) to the device. The aim of the time drift is compensating the time required for the request to arrive to the device, which is variable. 30 is a high value, the aim was compensating a possible delay in the network, but at the same time probably it's too much and we might miss a reboot. So, if you have a better value, happy to change it